openstates/mn/bills.py

```diff
@@ -157,47 +187,59 @@ def get_bill_info(self, chamber, session, bill_detail_url, version_list_url):
         doc = self.lxmlize(bill_detail_url)
 
         # Check if bill hasn't been transmitted to the other chamber yet
-        transmit_check = self.get_node(doc, '//h1[text()[contains(.,"Bills")]]/following-sibling::ul/li/text()')
-        if transmit_check is not None and 'has not been transmitted' in transmit_check.strip():
-            self.logger.debug('Bill has not been transmitted to other chamber ... skipping {0}'.format(bill_detail_url))
+        transmit_check = self.get_node(
+            doc,
+            '//h1[text()[contains(.,"Bills")]]/following-sibling::ul/li/text()'
+        )
+        if (transmit_check is not None and
+                'has not been transmitted' in transmit_check.strip()):
+            self.logger.debug('Bill has not been transmitted to other chamber '
+                              '... skipping {0}'.format(bill_detail_url))
             return
 
         # Get the basic parts of the bill
         bill_id = self.get_node(doc, '//h1/text()')
         self.logger.debug(bill_id)
-        bill_title_text = self.get_node(doc, '//h2[text()[contains(.,'
-            '"Description")]]/following-sibling::p/text()')
+        bill_title_text = self.get_node(
+            doc,
+            '//h2[text()[contains(.,"Description")]]/following-sibling::p/text()'
+        )
         if bill_title_text is not None:
             bill_title = bill_title_text.strip()
         else:
-            long_desc_url = self.get_node(doc, '//a[text()[contains(.,'
-                '"Long Description")]]/@href')
+            long_desc_url = self.get_node(
+                doc,
+                '//a[text()[contains(.,"Long Description")]]/@href'
+            )
             long_desc_page = self.lxmlize(long_desc_url)
             long_desc_text = self.get_node(long_desc_page, '//h1/'
-                'following-sibling::p/text()')
+                                           'following-sibling::p/text()')
             if long_desc_text is not None:
                 bill_title = long_desc_text.strip()
             else:
                 bill_title = 'No title found.'
                 self.logger.warning('No title found for {}.'.format(bill_id))
         self.logger.debug(bill_title)
-        bill_type = {'F': 'bill', 'R':'resolution',
+        bill_type = {'F': 'bill', 'R': 'resolution',
                      'C': 'concurrent resolution'}[bill_id[1]]
-        bill = Bill(session, chamber, bill_id, bill_title, type=bill_type)
+        bill = Bill(bill_id, legislative_session=session, chamber=chamber,
+                    title=bill_title, classification=bill_type)
 
         # Add source
         bill.add_source(bill_detail_url)
 
         # Add subjects.  Currently we are not mapping to Open States
         # standardized subjects, so use 'scraped_subjects'
-        bill['scraped_subjects'] = self._subject_mapping[bill_id]
+        # TODO: migrate to Pupa
+        # bill['scraped_subjects'] = self._subject_mapping[bill_id]
```

Comment:
correct- we don't have this distinction anymore, you can use bill.add_subject() to add these directly